src/components/popover/widgets/icon/icon.component.scss

```diff
@@ -2,3 +2,21 @@
     display: flex;
     align-items: center;
 }
+
+.action {
```

Comment:
Where are these styles used? Looking at the html they don't seem to match any element there.